pep-0701.rst

```diff
@@ -320,7 +336,75 @@ All restrictions mentioned in the PEP are lifted from f-string literals, as expl
+  strings are fully delimited by the same kind of quote, which by itself is a very simple rule. It
+  has been pointed out that one of the reasons quote reuse may be harder for humans to parse and
+  will lead to code that may be harder to read is that the quote character is the same on start and
+  end (as opposed to other delimiters). 
+
+* Some users have raised that quote reuse may break some lexer and syntax highlighting tools that rely
+  on simple mechanisms to detect strings and f-strings, such as regular expressions of simple delimiter
+  matching tools. Introducing quote reuse in f-strings will either make these tools more tricky to keep
+  working or will break them altogether (as for instances regular expressions cannot parse arbitrary nested
+  structures with delimiters). Some of the tools that may need some work to allow to syntax-highlight correctly
+  f-strings include the IDLE editor, which is included in the standard library.
+
+Here are some of the arguments in favour:
+
+* Many languages that allow similar syntactic constructs (normally called "string interpolation") allow quote
+  reuse and arbitrary nesting. Some of these languages include JavaScript, Ruby, C#, Bash, Swift and many others.
+  The fact that many languages allow quite reuse can be a compelling argument in favour of allowing it in Python. This
+  is because it will make the language more familiar to users coming from other languages. 
+
+* As many other popular languages allow quite reuse in string interpolation constructs, this means that editors
+  that support syntax highlighting for these languages will already have the necessary tools to support syntax
+  highlighting for f-strings with quote reuse in Python. This means that although the files that handle syntax
+  highlighting for Python will need to be updated to support this new feature, is not expected to be impossible
+  or very hard to do.
+
+* One advatange of allowing quote reuse is that it composes cleanly with other syntax. Sometimes this is referred
+  as “referential transparency”. An example of this is that if we have ``f(x+1)``, assuming a is a brand new variable, it
+  should behave the same as ``a = x+1; f(a)z``. And vice versa.  So if we have::
+
+    def py2c(source):
+        prefix = source.removesuffix(".py")
+        return f"{prefix}.c"
+
+  It should be expected that if we replace the variable prefix with its definition, the answer should be the same:
+
+    def py2c(source):
+        return f"{source.removesuffix(".py")}.c"
+
+* Limiting quote reuse will raise considerable the complexity of the implementation of the proposed changes. This is because
+  that will force the parser to have the context that is parsing an expression part of an f-string with a given quote in order
+  to know if it need to reject an expression that reuses the quite. Carrying this context around is not trivial in parsers that
```

Comment:
```suggestion
  to know if it needs to reject an expression that reuses the quote. Carrying this context around is not trivial in parsers that
```
The whole sentence is a bit long and complex, so it might be clearer to rewrite it.